lib/rtc_monitor.js

```diff
@@ -46,7 +46,7 @@ export class RTCMonitor extends EventEmitter {
         let totalJitter = 0;
         let totalLocalStats = 0;
         for (const [ssrc, stat] of Object.entries(localIn)) {
-            if (!this.stats.lastLocalIn[ssrc] || stat.timestamp <= this.stats.lastLocalIn[ssrc].timestamp) {
+            if (!this.stats.lastLocalIn[ssrc] || !this.stats.lastRemoteOut[ssrc] || stat.timestamp <= this.stats.lastLocalIn[ssrc].timestamp) {
```

Comment:
This is the real fix. It can happen that `lastRemoteOut` is not available yet.

Reply:
Did we encounter this happening? Or just preventative?

Reply:
@cpoile It happened this morning to @agarciamontoro

Reply:
Thanks for investigating this one!